pkg/skaffold/build/ko/builder.go

```diff
@@ -30,16 +31,16 @@ import (
 )
 
 func (b *Builder) newKoBuilder(ctx context.Context, a *latestV1.Artifact) (build.Interface, error) {
-	bo := buildOptions(a.KoArtifact.BaseImage, a.KoArtifact.Platforms, a.Workspace)
+	bo := buildOptions(a.KoArtifact.BaseImage, a.KoArtifact.Platforms, a.Workspace, a.KoArtifact.Dir)
 	return commands.NewBuilder(ctx, bo)
 }
 
-func buildOptions(baseImage string, platforms []string, workspace string) *options.BuildOptions {
+func buildOptions(baseImage string, platforms []string, workspace string, sourceDir string) *options.BuildOptions {
```

Comment:
It looks like `workspace` is unused here.  Maybe you could do the join in `newKoBuilder()` and rename the existing arg?

Reply:
The only reason I left it at two arguments was so I could cover the behavior with a unit test in `builder_test.go`. Admittedly it's a trivial test, but it serves the point of documenting the expected behavior.

I can move the call to `Join()` you prefer that though?